internal/rest/dequeue_nodes.go

```diff
@@ -260,6 +260,11 @@ func (rest *RestOperations) CheckAndPublishForRetry(err error, publishKey, key s
 					rest.PublishKeyToSlowRetryLayer(publishKey, key)
 					return true
 				}
+				if strings.Contains(*aviError.Message, lib.VrfContextNotFoundError) {
```

Comment:
Resource not found errors usually return 404.

Reply:
iirc, it returns 400 - Bad Request